modules/microshift-disconnected-host-con.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-disconnected-network-config.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-disconnected-host-preparation_{context}"]
+= Preparing networking for fully disconnected hosts
+
+Use the procedure that follows to start and run {microshift-short} clusters on devices running fully disconnected operating systems. A {microshift-short} host is considered fully disconnected if it has no addressable subnet.
```

Comment:
```suggestion
Use the procedure that follows to start and run {microshift-short} clusters on devices running fully disconnected operating systems. A {microshift-short} host is considered fully disconnected if it has no external network connectivity.
```

The subnet description might be a little obtuse for users.  This says (more or less) the same thing without insinuating the need to go and check for a subnet.